package.json

```
@@ -46,9 +46,9 @@
   "dependencies": {
     "chalk": "^4.1.0",
     "chokidar": "^3.5.1",
-    "commander": "^7.0.0",
+    "commander": "7.1.0",
     "cross-spawn": "^7.0.3",
-    "elm-tooling": "^1.1.0",
+    "elm-tooling": "1.2.0",
```

```suggestion
    "elm-tooling": "^1.2.0",
```

You can also run `npx elm-tooling tools` to interactively update Elm tools